Source/WebCore/Modules/streams/WritableStream.cpp

```diff
@@ -45,6 +46,23 @@ ExceptionOr<Ref<WritableStream>> WritableStream::create(JSC::JSGlobalObject& glo
     return create(globalObject, underlyingSinkValue, strategyValue);
 }
 
+WritableStream::~WritableStream() = default;
```

Comment:
cc @youennf to verify these unified build fixes are acceptable. They are appearing in https://github.com/WebKit/WebKit/pull/11154 too, I think these changes will fix them there as well